scripts/aem.js

```diff
@@ -621,6 +621,12 @@ function decorateBlock(block) {
     block.dataset.blockStatus = 'initialized';
     const blockWrapper = block.parentElement;
     blockWrapper.classList.add(`${shortBlockName}-wrapper`);
+    for (let i = 1; i < block.classList.length; i += 1) {
```

Comment:
do we ever change `aem.js`? I always thought this is off-limits :)

Reply:
We do change it, there should be no issue adding functionality here. 👍